install/base-cluster-roles.yaml

```diff
@@ -190,19 +190,37 @@ rules:
       - update
 ---
 
-## WIll be removed and replaced with RoleBinding for specific projects. 
-kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
 metadata:
-  name: runai-scheduler-rw
+  namespace: runai
+  name: runai-scheduler-runai-ns-only
```

Comment:
I would just change the name to something like "runai-scheduler-configmaps-editor" (ns-only is redundant because it's a Role (rather than ClusterRole), so obviously it's only inside a namespace)